ecowitt2mqtt/data.py

```diff
@@ -127,12 +129,20 @@ class DataProcessor:  # pylint: disable=too-many-instance-attributes
 
     _data: dict = field(init=False, repr=False)
     unique_id: str = field(init=False)
+    model: str = field(init=False)
+    stationtype: str = field(init=False)
 
     def __post_init__(self):
         """Set up some additional attributes from passed-in data."""
         object.__setattr__(
             self, "unique_id", self._input.pop("PASSKEY", DEFAULT_UNIQUE_ID)
         )
+        object.__setattr__(
+            self, "model", self._input.pop("model", DEFAULT_MODEL)
```

Comment:
Since we've introduced the `Device` class, should the model and station type be stored there, instead? Then we can pass around a single `Device`?